app/models/sample.rb

```diff
@@ -179,11 +179,24 @@ def initiate_input_file_upload
   def initiate_s3_cp
     return unless status == STATUS_CREATED
     stderr_array = []
+    total_reads_json = "total_reads.json"
```

Comment:
seems like convention is to put this into a frozen string

Reply:
thanks!  i often forget that

Reply:
so I moved it to the top as a frozen constant, and then I am not sure if I should do this
```
      total_reads_json_path = File.join(File.dirname(fastq.to_s), TOTAL_READS_JSON)
```
or add `.to_s` like so:
```
      total_reads_json_path = File.join(File.dirname(fastq.to_s), TOTAL_READS_JSON.to_s)
```
Definitely testing :)

Reply:
Btw another problem I had here with testing is that I can't figure out how to import File into the ruby shell `irb`.   I ended up just spinning up rails console as somehow it has that magically imported.

Reply:
The `.to_s` appears unnecessary.